src/AEOS/AEOSConnector/Connectors/AEOSConnector.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+
+namespace Innovatrics.SmartFace.Integrations.AEOSConnector.Connectors
+{
+    public class AEpuConnector : IAEOSConnector
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+        private readonly IHttpClientFactory httpClientFactory;
+
+        private System.Net.IPAddress ipAddr;
+
+        private System.Net.IPEndPoint localEndPoint;
+
+        public AEpuConnector(
+            ILogger logger,
+            IConfiguration configuration,
+            IHttpClientFactory httpClientFactory
+        )
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+            this.httpClientFactory = httpClientFactory ?? throw new ArgumentNullException(nameof(httpClientFactory));
+        }
+
+        public async Task OpenAsync(string AEpuHostname, int AEpuPort, string WatchlistMemberID)
+        {
+            this.logger.Information("Sending ipBadge to {AEpuHostname}:{AEpuPort} for user {WatchlistMemberID}", AEpuHostname, AEpuPort, WatchlistMemberID);
+
+            try 
+            {
+                // Establish the remote endpoint for the socket.
+
+                if (System.Net.Dns.GetHostAddresses(AEpuHostname).Length == 0)
```

Comment:
I would prefer consistent: either explicit call with fully qualified namespace everywhere or using in header everywhere

Reply:
changed to Dns.GetHostAddresses(AEpuHostname) instead and added using System.net